deps/rabbit/src/rabbit_deprecated_features.erl

```diff
@@ -346,6 +347,25 @@ get_warning(FeatureProps, Permitted) when is_map(FeatureProps) ->
             maps:get(when_removed, Msgs)
     end.
 
+-spec list(Which :: all | used) -> rabbit_feature_flags:feature_flags().
```

Comment:
Likewise, could you please name arguments and the return value, then define them below?